build/tasks/fonts/index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const chalk = require('chalk');
+const spawn = require('../../lib/spawn');
+const mkdir = require('../../lib/mkdir');
+
+module.exports = config => {
+  if (!config.fonts) {
+    return Promise.resolve();
+  }
+
+  return new Promise((resolve, reject) => {
+    fs.stat(config.fonts.src, err => err ? reject(err) : resolve());
+  })
+    .then(() => mkdir(config.fonts.out))
+    .then(() => spawn('cp', ['-r', config.fonts.src, config.fonts.out]))
+    .catch(e => {
+      if (e.code !== 'ENOENT') {
+        throw e;
```

Comment:
I'm trying to understand this, if you catch an error that doesn't have a specific path/ file directory, you'll throw an error?  Would it be better to provide a helpful error message?